src/app/buy-credits/buy-credits.component.scss

```diff
@@ -0,0 +1,61 @@
+@use '@angular/material' as mat;
+@import '../../abstract';
+
+#loggedInHeader {
+    background-color: green;
```

Comment:
I don't think we should be introducing brand new colours! Can we just use existing visual elements so this is consistent with the rest of the app while we test ahead of the new design?

Reply:
So get rid of the logged in header altogether?

Reply:
I think it's fine to keep it, looks good imo.

Reply:
Nice – yep I think giving it an element (until we have a new header component) makes sense – I just didn't want to introduce a mix and match of colour palettes that could muddy the rebrand launch.